EnglishFilter/sections/general_extensions.txt

```diff
@@ -235,6 +235,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/80479
+allviids.com#%#//scriptlet("abort-on-property-read", "checkLazyFreshPopunder")
```

Comment:
@wout-codes Is this rule still needed?
I can't find anything related to `checkLazyFreshPopunder` (in video player from `allviids.com`).

Reply:
@AdamWr Visit, `https://nerdolac.com/5/3744076`. Ad-server called by `allviids.com`.

Reply:
Okay, but it doesn't work on my end (I checked also a new rule). because `LazyFreshPopunder/checkLazyFreshPopunder` is not in global scope.
Something like this works fine on my side:
```
allviids.com#%#//scriptlet('prevent-setTimeout', 'smartOverlay')
allviids.com#%#//scriptlet('prevent-addEventListener', '', 'Popunder')
```

Reply:
You're talented. You know things I know not.